File: CORREDOR/4 Lenguajes/Go/Metodos_Parametros.go
package main

import "fmt"

func sumar(a int, b int) {
	resultado := a + b
	fmt.Println("La suma es:", resultado)
}

func restar(a int, b int) {
	resultado := a - b
	fmt.Println("La resta es:", resultado)
}

func areaTriangulo(base int, altura int) {
	resultado := (base * altura) / 2
	fmt.Println("El área del triángulo es:", resultado)
}

func promedio(a int, b int, c int) {
	resultado := a + b + c
	prom := resultado / 3
	fmt.Println("El promedio es:", prom)
}

func main() {
	sumar(8, 7)
	restar(18, 5)
	areaTriangulo(8, 6)
	promedio(9, 18, 12)
}

File: CORREDOR/4 Lenguajes/JS/Metodos_Parametros.js
class Sumar {
    sumar(a, b) {
        const resultado = a + b;
        console.log("La suma es: " + resultado);
    }
}

class Resta {
    restar(a, b) {
        const resultado = a - b;
        console.log("La resta es: " + resultado);
    }
}

class areaTriangulo {
    area(base, altura) {
        const resultado = (base * altura) / 2;
        console.log("El área del triángulo: " + resultado);
    }
}

class Promedio {
    inicial(a, b, c) {
        const resultado = a + b + c;
        const promedio = resultado / 3;
        console.log("El promedio es: " + promedio);
    }
}

const cal = new Sumar();
cal.sumar(8, 7);

const r1 = new Resta();
r1.restar(18, 5);

const area = new areaTriangulo();
area.area(8, 6);

const pro = new Promedio();
pro.inicial(9, 18, 12);